### ISSUE TITLE
fungal TB is not the hell disease of doom that it should be

### ISSUE THREAD

**Issue by hotimep** [BeeStation/BeeStation-Hornet#7794]
## Round ID & Round Date:
 2022-10-03, 40688

## Reproduction:
it was from a floorpill, supposedly.

HOWEVER, THIS BABY-ASS FUNGAL TB WAS SHIT!!! ALL IT DID WAS MAKE YOU THROW UP ALOT AND GIVE YOU MILD STAMINA DAMAGE!!! WHAT THE FUCK!!! FROM WHAT I REMEMBER IN THE OLD DAYS OF YORE (PRE-GOLDEN) FUNGAL TB IS SUPPOSED TO GIVE YOU NUCLEAR OXY DAMAGE AND SHRED YOUR LUNGS!!! AT THE COST OF BEING EASY TO CURE (AND COMING FROM NUKIES)!!! FIX THIS SHIT!!!

TL;DR IT SHOULD DO THIS
![fungal](https://user-images.githubusercontent.com/59359762/193513039-0bb9b3c2-6893-48c2-83db-41701cc11d63.png)

**Comment by etherware-novice** [comment]
ill do it

**Comment by hotimep** [comment]
GOOD

**Comment by PowerfulBacon** [comment]
I think the issue is that it doesn't spread via airborne transmission anymore

**Comment by zeskorion** [comment]
> I think the issue is that it doesn't spread via airborne transmission anymore

false. that change did not affect simple diseases like fungal tuberculosis

the real thing is that fungal tuberculosis is nonlethal past stage 4, and only causes heavy CC. It can kill at stage 4, but that is a very limited window. As far as I know, this is intentional behaviour- FTB is to incapacitate, and not kill, the crew. if it murdered everyone itd be a bit too potent IMO.

changing it to be deadly would be quite easy though

**Comment by hotimep** [comment]
IT COMES FROM NUKIES!!! (THEY ARE KNOWN TO KILL EVERYONE) IT SHOULD BE DEADLY!!!

**Comment by zeskorion** [comment]
(i will note beeboomers may confuse my old deathvirus i used to use constantly as traitor, which I named "viral supertuberculosis" may be confused for fungal tuberculosis. That virus actually did shred lungs through oxygen damage, and was often confused with FTB)
either way, testing a buff pr now, for if some maint thinks this is a good idea

**Comment by hotimep** [comment]
OTHER ISSUE MAKERS WISH THEY COULD HAVE THEIR ISSUES SOLVED IN RECORD TIME!!!

**Comment by zeskorion** [comment]
> OTHER ISSUE MAKERS WISH THEY COULD HAVE THEIR ISSUES SOLVED IN RECORD TIME!!!

I was tagged and this was literally 7 characters typed (,,  35[tab])

**Comment by hotimep** [comment]
THEY AREN'T GOOD LIKE ME!!!

**Comment by zeskorion** [comment]
> I think the issue is that it doesn't spread via airborne transmission anymore

(though you did request it be nerfed, so FTB does spread significantly slower now)
